Sources/GDSAnalytics/README.md

```diff
@@ -168,18 +218,35 @@ let event = FormEvent(textKey: "Did you start on a computer or tablet?",
 | text | The text of the question that was asked in the form. | Lowercase. Limited to 100 characters. | `did you start on a computer or tablet?` |
 | response | The response to the form that the user entered. | Lowercase. Limited to 100 characters. | `yes, I started on a computer or tablet` |
 
-### Popup Alerts
 
-This type of event should be logged when a call-to-action is displayed, usually this would be a `UIAlertController` but it may also be used for other modal inputs. Use the `PopupEvent` type for this.
+### Form Action Menu Event 
+
+This type of event fires once a users repsonse to a form or question is locked in i.e. as they navigate from one page to the next.
 
 ```swift
-let event = PopupEvent(textKey: "Change your permissions")
+let event = FormActionMenuEvent(textKey: "question answer")
 ```
 
 #### Expected Parameters:
 
 | Parameter | Description | Expected Format | Example |
 | ------------- |-------------| -----| --- |
-| name | The name of the event that occured, for popups this is `popup`. | N/A | `popup` |
-| type | The type of event that occured, for popups this is `call to action`. | N/A | `call to action` |
-| text | The text that is displayed on the popup. | Lowercase. Limited to 100 characters. | `change your permissions` |
+| name | The name of the event that occured, for popups this is `formResponse`. | N/A | `formResponse` |
+| type | The type of event that occured, for alerts this is `actionMenu`. | N/A | `actionMenu` |
+| text | The text that is displayed on the popup. | Lowercase. Limited to 100 characters. | `question answer` |
+
+### Form CTA Event 
+
+This type of event fires once a users response to a form or question is locked in i.e. as they navigate from one page to the next.
```

Comment:
This description is the same as `FormActionMenuEvent` which poses the question of how these two types are different?

Reply:
Ah okay, is this for an actual form event whereas above is for an action sheet on a form?

Reply:
Its a fair question, I've asked Sam as the descriptions are the same for these 2 events 😅

Reply:
I have updated per Sam's description.